docs/resources/control-flow/user-interactivity/forms/form-widgets/dropdown.md

```diff
@@ -116,7 +116,7 @@ To set an initial value:
 
 You might want to save the dropdown value as soon as the selection changes. This approach is useful when you want to ensure that the user's selection is immediately saved without having to wait for them to submit the form. By doing so, you can provide a better user experience and reduce the risk of data loss in case of any interruption.
 
-You can do so by adding an action such as [update app state](#), [update Firestore record](#) that [triggers when a selection changes](#) on this widget.
+You can do so by adding an action such as [update app state](../../../../../resources/data-representation/app-state/#update-app-state-variable), [update Firestore record](../../../../../integrations/database/cloud-firestore/firestore-actions/#update-document-action) that [triggers when a selection changes](../../../../../resources/ui-building-blocks/widgets/widget-commonalities/#trigger-action-on-selection-change) on this widget.
```

Comment:
There should be a / before # in header references, this will throw an error in IDE.